images/rootfs.yml.in

```diff
@@ -12,6 +12,7 @@ init:
   - FW_TAG
   - XEN_TAG
   - GPTTOOLS_TAG
+  - KEXECTOOLS_TAG
```

Comment:
Will check that.

Reply:
Works with the following patch:
```
diff --git a/images/rootfs-mini.yml.in.patch b/images/rootfs-mini.yml.in.patch
index 55db9fdece65..a25cbd3dd8f4 100644
--- a/images/rootfs-mini.yml.in.patch
+++ b/images/rootfs-mini.yml.in.patch
@@ -1,6 +1,6 @@
 --- images/rootfs.yml.in       2022-06-24 08:41:44.000000000 +0300
 +++ images/rootfs-mini.yml.in  2022-06-24 08:45:15.000000000 +0300
-@@ -1,74 +1,18 @@
+@@ -1,75 +1,19 @@
  kernel:
 -  image: KERNEL_TAG
 +  image: NEW_KERNEL_TAG
@@ -16,6 +16,7 @@
 -  - FW_TAG
 -  - XEN_TAG
    - GPTTOOLS_TAG
+   - KEXECTOOLS_TAG
  onboot:
 -   - name: rngd

```

Will include this to the next iteration.

Reply:
I think it would be reasonable to remove the package from the mini version for now.

Reply:
Ok, what is the purpose of the mini version, btw?

Reply:
We [build](https://github.com/lf-edge/eve/blob/26fb708b5dbe96c7bb0c0bb0fef2b86653a02d4b/.github/workflows/publish.yml#L124) this version for riscv64, because not all packages are ready for this arch.